src/gatsby/src/components/park/nearbyParks.js

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+import NearbyPark from "./nearbyPark"
+
+const NearbyParks = ({ parks }) => {
+  const data = useStaticQuery(graphql`
```

Comment:
Can you please try getting CMS-62 done first before we merge this so we don't have to get back a list of all the photos for all the parks on every park page.

Reply:
I think this can be updated now that CMS-62 has been merged